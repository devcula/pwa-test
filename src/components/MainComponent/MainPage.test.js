// import { mount } from 'enzyme';
// import toJson from 'enzyme-to-json';
// import App from './App';
// import React from 'react';


test("it should render App component", () => {
    // const wrapper = mount(<App />);
    // expect(toJson(wrapper)).toMatchSnapshot();
})