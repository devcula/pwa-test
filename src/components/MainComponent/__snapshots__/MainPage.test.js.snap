// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render MainPage component will null values 1`] = `
<MainPage
  robots={Array []}
  searchField=""
>
  <h1
    className="tc"
  >
    LOADING...
  </h1>
</MainPage>
`;
